docs/get_started/quickstart.mdx

```diff
@@ -532,6 +532,136 @@ go build .
 
 </details>
 
+</TabItem>
+<TabItem value="python" label="Python" default>
+
+        <Admonition type="note" title="Prerequisites">
+            - Python >= v3.11
+            - Optional but recommended: [Install the Restate Server & CLI](/develop/local_dev#running-restate-server--cli-locally)
+        </Admonition>
+
+        <Step stepLabel="1" title="Get the Greeter service template">
+
+            <CH.Code>
+                ```shell CLI
+                # link(1:3) /develop/local_dev#running-restate-server--cli-locally
+                restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell npx
+                npx @restatedev/restate example python-hello-world &&
+                cd python-hello-world
+                ```
```

Comment:
Showing npx is very weird here, i'm a python user i know nothing about npm :)

Reply:
I show it everywhere, it's up for the user to decide on what he has installed...